docs/howto/garbage-collection-index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Garbage Collection
+description: Clean up unnecessary objects using the garbage collection feature in lakeFS.
+parent: How-To
+nav_order: 20
+has_children: true
+redirect_from: ../reference/garbage-collection.html
+---
```

Comment:
Maybe we should give a summary and highlight the managed solution here?

Reply:
See other comment